charts/opentelemetry-collector/values.yaml

```diff
@@ -500,6 +500,8 @@ prometheusRule:
   # Create default rules for monitoring the collector
   defaultRules:
     enabled: false
+    # additional labels for rules (to be used with alert manager routing)
+    extraLabels: {}
```

Comment:
I am a little torn on this as you could achieve your desired outcome via the `groups` field.  Typically in the charts we like to keep the opinionated, out-of-the-box configuration simple, excluding additional configuration. @dmitryax @Allex1  @povilasv what do you think?

At the minimum we should use a different name since this name conflicts with the other `extraLabels` field which is adding k8s labels.

Reply:
If I was to use `groups` I would have to re-write all default rules, for sake of adding some extra labels.

Now that you've mentioned it, `extraLabels` does seem like suboptimal name.
Maybe:
- `extraRuleLabels`
- `ruleLabels`
- `additionalRuleLabels` - this one is used by kube-prometheus-stack chart

I would also have to make it explicit, that `severity` label is pre-defined.

Reply:
I get your point, yet there are other big projects out there (like kube-prometheus-stack) that do allow for that, and actually have full blown customization in that area.
It would be great not to ask teams to copy-paste whole huge rule definitions for sake of adding 2-3 labels :/

Reply:
Would be nice if this feature was accepted, there is however a workaround in place, so I will be able to achieve my goal. Unfortunately it will be suboptimal, and will require more maintenance, yet it will work :)